css/app.css

```diff
@@ -410,8 +410,9 @@ header.search-active .search-navigation-button {
   display: flex;
   flex: 1;
   font-size: 15px;
-  justify-content: center;
+  justify-content: left;
```

Comment:
I think we still want the filename centered actually. We need to figure out why it's overlapping with the other elements and stop that.